hoodie-client/src/main/java/com/uber/hoodie/config/HoodieStorageConfig.java

```diff
@@ -44,6 +44,9 @@
   public static final String PARQUET_COMPRESSION_RATIO = "hoodie.parquet.compression.ratio";
   // Default compression ratio for parquet
   public static final String DEFAULT_STREAM_COMPRESSION_RATIO = String.valueOf(0.1);
+  public static final String PARQUET_COMPRESSION_CODEC = "hoodie.parquet.compression.codec";
+  // Default compression ratio for parquet
+  public static final String DEFAULT_STREAM_COMPRESSION_CODEC = "gzip";
```

Comment:
nit: consistently name to `DEFAULT_PARQUET_COMPRESSION_CODEC" ?

Reply:
I will change it, thx.